bigbluebutton-html5/imports/ui/components/chat/service.js

```diff
@@ -152,7 +152,6 @@ const sendMessage = (receiverID, message) => {
     fromTimezoneOffset: (new Date()).getTimezoneOffset(),
```

Comment:
Should we also move `fromTimezoneOffset` to server side?

Reply:
@antobinary Then we will have the timezoneOffset of the server not the client :)